packages/aws-fargate/src/index.js

```diff
@@ -5,6 +5,16 @@
 
 'use strict';
 
+const minimumNodeJsVersion = require('@instana/core/src/util/minimumNodeJsVersion');
+if (require('@instana/core/src/util/nodeJsTooOld')(minimumNodeJsVersion)) {
```

Comment:
I have bundled both the minimum version and the check into one file, thereby we only have one import now. I left the console.error in each entry file because the text is a bit different for each package.